src/sensors/rev_encoder.hpp

```diff
@@ -1,16 +1,58 @@
+/**
+ * @file rev_encoder.hpp
+ * @brief The header file for the Rev Encoder firmware
+ * @date 2023-10-31
+ */
+
+#include <cmath>
 #include <FreqMeasureMulti.h>
 
 #ifndef REV_ENCODER_H
 #define REV_ENCODER_H
 
-class RevEncoder {
+/**
+ * @brief the class for the Rev Encoder
```

Comment:
Can you describe what a "rev encoder" is instead? What is it, why is it useful? 
Also, please use triple /// instead for all doxygen comments

Reply:
added a link to the exact item

Reply:
Also, it's worth including a note about the exact parts/model numbers we use in the robots